src/libraries/System.Reflection.Metadata/src/System/Reflection/Internal/MemoryBlocks/StreamMemoryBlockProvider.cs

```diff
@@ -133,7 +133,7 @@ private unsafe bool TryCreateMemoryMappedFileBlock(long start, int size, out Mem
                 // CreateMemoryMap might modify the stream (calls FileStream.Flush)
                 lock (_streamGuard)
                 {
-                    newMemoryMap = MemoryMapLightUp.CreateMemoryMap(_stream);
+                    newMemoryMap = MemoryMapLightUp.CreateMemoryMap(_stream!);
```

Comment:
`_stream` doesn't need to be nullable as always init-ed and only nulled on dispose as mentioned before